meta-lmp-base/recipes-graphics/wayland/weston-init.bbappend

```diff
@@ -15,4 +17,9 @@ do_install_append_lmp-wayland() {
 
     install -m 0644 ${WORKDIR}/utilities-terminal.png ${D}${datadir}/weston/icon/utilities-terminal.png
     install -m 0644 ${WORKDIR}/background.jpg ${D}${datadir}/weston/backgrounds/background.jpg
+    install -Dm 0644 ${WORKDIR}/weston.conf ${D}${sysconfdir}/default/weston
```

Comment:
There is already a weston.env file doing exactly what you need here, so instead of creating a new file and installing it here, just create a custom weston.env instead (which will then be used by the main recipe).

```
meta/recipes-graphics/wayland/weston-init.bb:           file://weston.env \
meta/recipes-graphics/wayland/weston-init.bb:   install -Dm644 ${WORKDIR}/weston.env ${D}${sysconfdir}/default/weston
```